tests/venv/newline-conversion/postBuild

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# I contain CRLF line endings on purpose

```

Comment:
I had hoped to see some extra characters/symbol here on GH to verify that this file contains CRLF. Could someone on a not-windows system checkout this branch to confirm my `.gitattribute` worked?

Reply:
I opened it on my windows machine and it said that it was in CRLF mode, which is what we want. Just FYI, you should be able to open this with Atom or something like this, and it should tell you the line endings present in the file (and if not by default, then there are plugins for this kinda thing for most text editors)

Reply:
Locally I see the extra characters but because git can/will mangle the line endings when you checkin/-out I wanted to see if someone somewhere else can also see them after it has gone through git :)